### PR TITLE
Assignment 2.2_Vorobeva M

### PR DESCRIPTION
**Pull request by Murwar**
Была натренирована с помощью PyTorch небольшая модель, состоящая из трех full-connected слоев и трех ReLU. Для оптимизации добавлена Batch Normalization, вместо SGD используется AdamW. Точность на тестовой выборке составляет 0.8183.

### CODE REVIEW

**Comment by Murwar** [comment]
@toshiks
